nailgun/entities.py

```diff
@@ -2945,6 +2945,7 @@ class Host(  # pylint:disable=too-many-instance-attributes
 
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
+            'all_parameters': entity_fields.DictField(),
```

Comment:
Actually that's a list of parameters but since parameter require some parent entity to be specified for initialization and that entity is not returned here - no other way, unfortunately, it'll be test responsibility to pass right parent to be able to init parameter.